components/HamburgerMenu.vue

```diff
@@ -183,14 +183,28 @@
                         data-testid="hamburger-menu-footer-copyright"
                         class="text-primary-text-muted text-xs"
                     >
-                        © {{ new Date().getUTCFullYear() }} {{ $t('hamburgerMenu.copyright') }}
-                        <div class="mt-0.5">
-                            NPO
-                            <NuxtLink
-                                :to="'https://www.npo-hiroba.or.jp/search/zoom.php?pk=121289'"
-                                target="_blank"
-                                class="underline"
-                            >#9011005010215</NuxtLink>
+                        <div>
+                            © {{ new Date().getUTCFullYear() }} {{ $t('hamburgerMenu.copyright') }}
+                        </div>
+                        <div
+                            data-testid="hamburger-menu-footer-legal"
+                            class="flex"
+                        >
+                            <span class="mt-0.5">
+                                NPO
+                                <NuxtLink
+                                    :to="'https://www.npo-hiroba.or.jp/search/zoom.php?pk=121289'"
+                                    target="_blank"
+                                    class="underline"
+                                >#9011005010215</NuxtLink>
+                            </span>
+                            <span class="ml-2 mt-0.5">
+                                <NuxtLink
+                                    :to="'https://www.npo-hiroba.or.jp/search/zoom.php?pk=121289'"
+                                    target="_blank"
+                                    class="underline"
+                                >{{ $t('hamburgerMenu.balancesheet') }}</NuxtLink>
```

Comment:
Our linter didn't want the closing tag on a separate line? 🤔 Interesting

Reply:
nope 🤔 but I'll move it